CMakeLists.txt

```diff
@@ -230,6 +231,11 @@ if(NOT DEFINED SNMALLOC_ONLY_HEADER_LIBRARY)
 
   if(NOT WIN32)
     set(SHARED_FILES src/override/new.cc src/override/malloc.cc)
+    if (SNMALLOC_STATIC_LIBRARY)
+      add_shim(snmallocshim-static STATIC ${SHARED_FILES})
```

Comment:
Should the static lib be compiling `new.cc`  this is overriding the `new` and `delete` operations globablly without namespace?